contracts/Comptroller/ComptrollerStorage.sol

```diff
@@ -235,7 +235,7 @@ contract ComptrollerV10Storage is ComptrollerV9Storage {
 }
 
 contract ComptrollerV11Storage is ComptrollerV10Storage {
-    /// @notice Whether the delegate is allowed to borrow on behalf of the borrower
+    /// @notice Whether the delegate is allowed to borrow or redeem on behalf of the user
     //mapping(address borrower => mapping (address delegate => bool approved)) public approvedDelegates;
```

Comment:
In the context of VPB-01, I think we should update this comment too:

```
//mapping(address user => mapping (address delegate => bool approved)) public approvedDelegates;
```

Reply:
Done https://github.com/VenusProtocol/venus-protocol/pull/442/commits/becfe891329b8c93f46e968051721848d6d05253